client/modules/posts/viewPost/viewPost.js

```diff
@@ -17,30 +25,25 @@ Template.viewPost.events({
 		if(!this.text){
 			return;
 		}
-
-		post = {
-			target: e.currentTarget,
-			text: this.text,
-			markupText: this.text
-		}
-
+		Session.set('activePost', this._id);
 	},
 
 	'mouseleave .post': function(e) {
-		post = {};
+		Session.delete('activePost');
 	},
 
 	'mouseenter .tag a': function(e) {
-		if(!post.text){
+		var activePost = Posts.findOne(Session.get('activePost'));
```

Comment:
Det er interessant. Da er jeg enig i at det ikke er vits. Kanskje vi bør finne en quote fra dokumentasjonen slik at vi har det svart-på-hvitt om vi skal få spørsmål. For det ser litt rart ut å gjøre et databaseoppslag på hver mouseover.